src/org/intellij/erlang/psi/impl/ErlangPsiImplUtil.java

```diff
@@ -1873,4 +1881,101 @@ private boolean expectedFunction(@Nullable PsiElement resolve) {
       return true;
     }
   }
+
+  public static class ErlangFunctionCallNamedParameter<T extends PsiElement> extends PatternCondition<T> {
+    private final List<String> myNames;
+
+    public ErlangFunctionCallNamedParameter(@NotNull String... names) {
+      super("functionCallNamedParameter");
+      myNames = ContainerUtil.newArrayList(names);
+    }
+
+    @Override
+    public boolean accepts(@NotNull T element, ProcessingContext context) {
+      ErlangExpression expr = PsiTreeUtil.getParentOfType(element, ErlangExpression.class);
+      ErlangArgumentList args = ObjectUtils.tryCast(expr != null ? expr.getParent() : null, ErlangArgumentList.class);
+      if (expr == null || args == null) return false;
+
+      int position = args.getExpressionList().indexOf(expr);
+      if (position == -1) return false;
+
+      ErlangFunctionCallExpression call = ObjectUtils.tryCast(args.getParent(), ErlangFunctionCallExpression.class);
+      ErlangFunction function = resolveToFunction(call);
+      if (function == null) return false;
+
+      for (ErlangFunctionClause clause : function.getFunctionClauseList()) {
+        if (myNames.contains(getParameterName(clause, position))) return true;
+      }
+      return false;
+    }
+
+    @Nullable
+    private static String getParameterName(@NotNull ErlangFunctionClause clause, int position) {
+      List<ErlangArgumentDefinition> definitions = clause.getArgumentDefinitionList().getArgumentDefinitionList();
+      ErlangArgumentDefinition definition = definitions.size() > position ? definitions.get(position) : null;
+      ErlangMaxExpression expression = ObjectUtils.tryCast(definition != null ? definition.getExpression() : null,
+                                                           ErlangMaxExpression.class);
+      ErlangQVar var = expression != null ? expression.getQVar() : null;
+      return var != null ? var.getName() : null;
+    }
+  }
+
+  public static class ErlangFunctionCallModuleParameter extends PatternCondition<ErlangQAtom> {
+    public ErlangFunctionCallModuleParameter() {
+      super("functionCallModuleParameter");
+    }
+
+    @Override
+    public boolean accepts(@NotNull ErlangQAtom atom, ProcessingContext context) {
+      ErlangMaxExpression expr = ObjectUtils.tryCast(atom.getParent(), ErlangMaxExpression.class);
+      ErlangArgumentList args = ObjectUtils.tryCast(expr != null ? expr.getParent() : null, ErlangArgumentList.class);
+      ErlangSpecification specification = getSpecification(args);
+      if (expr == null || args == null || specification == null) return false;
```

Comment:
I've already made checking of all these parents in ErlangReferenceContributor, but I need these variables below. So I've retained them.